pkg/resourcetopologyexporter/resourcetopologyexporter.go

```diff
@@ -32,11 +33,16 @@ type Args struct {
 	TimeUnitToLimitEvents  time.Duration
 }
 
+type Handle struct {
+	PodResCli podresourcesapi.PodResourcesListerClient
+	K8SCli    kubernetes.Interface
+}
+
```

Comment:
This is not great indeed. The definition of this struct doesn't really belong in either (nor in `podres` or `k8shelper` package unfortunately). I'll try to think of a better place and try to avoid duplication.